plugins/com.google.cloud.tools.eclipse.appengine.libraries.test/src/com/google/cloud/tools/eclipse/appengine/libraries/model/MavenCoordinatesTest.java

```diff
@@ -159,6 +160,7 @@ public void testSetEmptyClassifier() {
         .setArtifactId("a")
         .setClassifier("")
         .build();
-    assertThat(mavenCoordinates.getClassifier(), is(""));
+    assertTrue(mavenCoordinates.getClassifier().isEmpty());
```

Comment:
Was there a reason for this change?  The original comparison to "" will cause JUnit to report the classifier string value, whereas this change will only report that it was not true.